ci/repository-request-access.md

```diff
@@ -0,0 +1,121 @@
+---
+title: CoP Repositories Request and Access
```

Comment:
What about the Galaxy organization, shouldn't it be there as well, even if there is no automation yet?

Reply:
Added a section on Galaxy